aws/resource_aws_ec2_client_vpn_network_association.go

```diff
@@ -29,14 +30,19 @@ func resourceAwsEc2ClientVpnNetworkAssociation() *schema.Resource {
 			},
 			"security_groups": {
 				Type:     schema.TypeSet,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+				Optional: true,
 				Computed: true,
```

Comment:
This attribute should include some extra resource documentation since we will want to leave it `Computed: true` to handle the case of the automatically configured default VPC security group, but we should specifically call out the behavior that to remove all custom security groups from the Terraform configuration, it must remain configured with at least one security group (e.g. the default VPC security group).